db/migrate/20180529160537_update_completed_ephemera_folders.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+class UpdateCompletedEphemeraFolders < ActiveRecord::Migration[5.1]
+  # Perform the one-way migration for updating EphemeraFolders
+  def change
+    EphemeraFolderMigrator.call
```

Comment:
If we bake this service and method into the DB migration, do we then have to keep it around forever? Unless we are going to use the EphemeraFolderMigrator more than once, I wonder if it makes sense to create a migrate job that we can later remove when it's no longer needed.

Reply:
That's a very fair point.  This approach was structured following the implementation for the migration to Valkyrie 1.0:
https://github.com/pulibrary/figgy/blob/master/db/migrate/20180306193020_migrate_to_valkyrie1.rb#L2

I can move the functionality directly into the migration and remove the service Class if that is preferred.

Reply:
Maybe this is a good point for discussion about best practices related to data migration. I have the same concern about that valkyrie1 service. Over time, these will pile up in the code base. We should collectively decide if we are good with that.